nemo/collections/tts/modules/fastpitch.py

```diff
@@ -267,6 +282,25 @@ def forward(
 
         enc_out = enc_out + pitch_emb.transpose(1, 2)
 
+        # Predict energy
+        if self.energy_predictor is not None:
+            energy_pred = self.energy_predictor(enc_out, enc_mask).squeeze(-1)
+
+            if energy is not None:
+                # Average energy over characters
+                energy_tgt = average_pitch(energy.unsqueeze(1), attn_hard_dur)
```

Comment:
Would supporting the situation when learn_alignment is false only required changing "attn_hard_dur" to "durs"? Or are these variables in different formats?

Reply:
Made the change. Although it added one more if...else. I think we should discuss about if we want to support not learning aligner anymore.

Reply:
Done.